### DISCUSSION TITLE
Is there a way to change an admin username?

### DISCUSSION THREAD

**Discussion by zikasak** [tobychui/zoraxy#441] [Q&A] [answered]
Title

**Comment by tobychui** [comment, marked as answer]
Through the UI? no.

Still, you can do that using tools like [boltbrowser](https://github.com/br0xen/boltbrowser) to open the `sys.db` and modify the auth bucket json content. Not the most elegant way to do it but it should work.

**Reply by zikasak** [reply]
Thanks! it worked
